solutions/observability/observability-serverless-feature-tiers.md

```diff
@@ -0,0 +1,57 @@
+---
+navigation_title: Serverless feature tiers
+applies_to:
+  serverless: ga
+products:
+  - id: observability
+---
+
+# {{obs-serverless}} feature tiers
```

Comment:
worried about separating this info from the other licensing info - would you consider keeping it with the other section?

Reply:
Do you mean moving this page as a whole to a different section, or breaking up the information across billing dimensions/ech-serverless comparison/project settings?

Reply:
moving this page from the observability section to either of the docs in the deploy/manage section. 

I realize neither page might be suitable in its current form, but we were asked to keep all of the licensing/tiering info together at some point so this moves away from that. I also am not sure whether it's an obs user decision as much as an admin/evaluator decision

Reply:
Yes @mdbirnstiehl, but I think this requires a bit more thinking, it shouldn't block this PR from moving forward. I've been thinking about this too and about more alignment between how we treat similar security and obs topics. We can sync separately on that.